src/components/header/contact-us-popover/index.tsx

```diff
@@ -9,27 +9,27 @@ import {
     contactUsPopoverImageStyles,
     contactUsPopoverContainerStyles,
 } from "../styles";
+import { useState } from "react";
 
-interface contactUsPopoverProps {
-    openContactUSPopoverButton: HTMLButtonElement | null;
-    setOpenContactUSPopoverButton: (element: HTMLButtonElement | null) => void;
-}
+const ContactUsPopover = () => {
+    const [openContactUSPopoverButton, setOpenContactUSPopoverButton] =
+        useState<HTMLButtonElement | null>(null);
+
+    const [openAndClosePopover, setOpenAndClosePopover] = useState(false);
 
-const ContactUsPopover = ({
-    openContactUSPopoverButton,
-    setOpenContactUSPopoverButton,
-}: contactUsPopoverProps) => {
     const handleOpenContactUsPopover = (
         event: React.MouseEvent<HTMLButtonElement>
     ) => {
         setOpenContactUSPopoverButton(event.currentTarget);
+        setOpenAndClosePopover(!openAndClosePopover);
     };
 
     const handleCloseContactUsPopover = () => {
         setOpenContactUSPopoverButton(null);
+        setOpenAndClosePopover(false);
     };
 
-    const isPopoverOpen = Boolean(openContactUSPopoverButton);
+    const isPopoverOpen = Boolean(openAndClosePopover);
     const contactUsPopoverId = isPopoverOpen ? "simple-popover" : undefined;
 
     const anchorOriginVerticalCenterHorizontalBottom = {
```

Comment:
consider moving anchorOriginVerticalCenterHorizontalBottom and transformOriginVerticalCenterHorizontalTop outside of the component to make the code cleaner.